react-form-controls/src/index.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+
+class RegistrationForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { username: '', password: '' };
+    this.handleUsernameChange = this.handleUsernameChange.bind(this);
+    this.handlePasswordChange = this.handlePasswordChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleUsernameChange(event) {
+    this.setState({ username: event.target.value });
+  }
+
+  handlePasswordChange(event) {
+    this.setState({ password: event.target.value });
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    console.log(this.state);
+
+  }
+
+  render() {
+
+    return (
+      <form>
+        <label>Username</label>
+        <input value={this.state.username} onChange ={this.handleUsernameChange}></input>
+        <label>Password</label>
+        <input type="password" value={this.state.password} onChange ={this.handlePasswordChange}></input>
+        <button onClick={this.handleSubmit}>Sign Up</button>
```

Comment:
`handleSubmit` should be on the `<form>` element on `onSubmit`, not directly on the button:

```JSX
<form onSubmit={this.handleSubmit}>
```